syscall.go

```diff
@@ -178,21 +195,119 @@ func waitForBlockDevice(deviceName string) error {
 	return nil
 }
 
-func mountContainerRootFs(containerID, image, rootFs, fsType string) (string, error) {
+var scsiHostPath = "/sys/class/scsi_host"
+
+// scanSCSIBus scans SCSI bus for the given SCSI address(SCSI-Id and LUN)
+func scanSCSIBus(scsiAddr string) error {
+	if _, err := os.Stat(scsiHostPath); err != nil {
```

Comment:
This call isn't needed as the `ReadDir()` below will detect `ENOENT` itself. Same comment for the `Stat()` in `findSCSIDisk()`.